architecture/design/ysql-gin-indexes.md

```diff
@@ -0,0 +1,107 @@
+# YSQL GIN indexes
+
+As of v2.6/v2.7, Yugabyte supports only one type of index access method for DocDB-backed relations: `lsm`.  This simply maps columns to the primary key of the base table.
```

Comment:
And just do a global search and replace for `.  ` (dot space space) and replace with a single space after the period `. `